app/helpers/patient_query_helper.rb

```diff
@@ -470,6 +497,87 @@ def patients_by_field_timeframe(patients, field, timeframe, tz_diff, type)
 
     patients
   end
+
+  def linelist(patients, workflow, tab)
```

Comment:
Same as my other comment about here versus in the controller itself.

Reply:
See above comment - will resolve this however we resolve the above.